src/unix/linux/cpu.rs

```diff
@@ -828,7 +828,7 @@ pub(crate) fn get_vendor_id_and_brand() -> HashMap<usize, (String, String)> {
             while let Some(line) = lines.next() {
                 if line.starts_with("vendor_id\t") {
                     info.vendor_id = Some(get_value(line));
-                } else if line.starts_with("model name\t") {
+                } else if line.starts_with("model name\t") || line.starts_with("Model Name\t") {
```

Comment:
What do you think about this instead:

```suggestion
                } else if line.len() > 11 && line.as_bytes().eq_ignore_ascii_case("model name\t") {
```

We can even extend it to other checks.

Reply:
Done

Reply:
> What do you think about this instead:

I prefer to use `split_once` in conjunction with `trim` on a one-off basis, as it were:

```rust
                match line.split_once(":") {
                    Some((k, v)) if k.trim().eq_ignore_ascii_case("vendor_id") => {
                        info.vendor_id = Some(k.trim().to_string())
                    }
                   ...
```

Reply:
I didn't want to use `trim` in case the name is "extended", like `cpu implementer id` which would match `cpu implementer` whereas it shouldn't. So please remove the `trim` call and keep the `\t` at the end of the name we're checking.

Reply:
Forgot to mention but `split_once` is a very good idea!